src/librustc_resolve/lib.rs

```diff
@@ -949,6 +928,9 @@ pub struct NameBinding<'a> {
     modifiers: DefModifiers,
     kind: NameBindingKind<'a>,
     span: Option<Span>,
+    // Enum variants are always considered `PUBLIC`, this is needed for `use Enum::Variant`
+    // or `use Enum::*` to work on private enums.
```

Comment:
This comment is no longer true after https://github.com/rust-lang/rust/pull/32875/commits/d1ef3561305cb43071c34fbbfad8ecc649996889?